docs/events/gsoc/2025/kotlin-code-quality-with-problems-api.md

```diff
@@ -2,10 +2,20 @@
 
 This project aims to implement a unified and actionable code quality reporting system through the integration of the recently introduced Gradle new [Problems API](https://docs.gradle.org/current/userguide/reporting_problems.html). This API provides structured and rich information about the problems happening during a build. The information provided can be used by different interfaces such as Gradle’s console output, build scans, or IDEs to communicate any issues. With this API integration and structured format of the generated report, developers can experience faster issue identification, streamline debugging, and have improved code maintainability. This project will provide value to the Kotlin development community by upgrading the quality of their error reporting. The integration of the Problems API will done in the plugins: Detekt and Ktlint.
 
+<head>
```

Comment:
You should not need it for the Material for MkDocs site.
Instead of that, you can use the front matter like in https://github.com/gradle/community/blob/main/docs/events/README.md (for text) 

Images are more complicated for the FOSS version of Materials for MkDocs though